### ISSUE TITLE
508: Test Summary table not in Aria Live

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#4831]
Issue:
In the Export functionality, the Test Summary table and 'There are no records to display' message should be in an aria-live region

Steps to Recreate:
- Login to ECMPS
- Navigate to Export in the left-hand menu, click on it, select and open a Configuration
- Check QA & Certification and Preview data
- The Test Summary data when displayed is not in the aria-live region

Also when there is no data to display for the selections, the message that is displayed is not in the aria-live region

**Comment by aprematta** [comment]
The no records message is now is an aria live but the tables are not

**Comment by aprematta** [comment]
On closer inspection, the aria live is present in the html..... it was not showing up via ANDI
